lib/whitehall_importer/integrity_checker.rb

```diff
@@ -58,6 +58,25 @@ def content_problems
       problems
     end
 
+    def time_problems
+      problems = []
+
+      proposed_first_published_at = proposed_payload["first_published_at"]
+      first_published_at = publishing_api_content["details"]["first_public_at"]
+
+      unless time_matches?(proposed_first_published_at, first_published_at)
+        problems << problem_description("first_published_at doesn't match",
```

Comment:
Might want to adjust to reflect that it doesn't match first_public_at eg "our first_published_at doesn't match first_public_at" ?

Reply:
Sure